sys-boot/udk/udk-2018.ebuild

```diff
@@ -0,0 +1,256 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 )
+PYTHON_REQ_USE="sqlite"
+
+inherit flag-o-matic multiprocessing python-single-r1 toolchain-funcs
+
+DESCRIPTION="Tianocore UEFI Development kit"
+HOMEPAGE="https://github.com/tianocore/tianocore.github.io/wiki/EDK-II"
+
+if [[ ${PV} == *9999* ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/tianocore/edk2.git"
+	KEYWORDS=""
+else
+	MY_V="UDK$(ver_cut 1)"
+	SRC_URI="https://github.com/tianocore/edk2/archive/v${MY_V}.tar.gz -> ${P}.tar.gz
+	doc? ( https://github.com/tianocore/edk2/releases/download/v${MY_V}/${MY_V}.Documents.zip -> ${P}-docs.zip )"
+	S="${WORKDIR}/edk2-v${MY_V}"
+	KEYWORDS="~amd64 ~x86"
+fi
+
+LICENSE="BSD-2"
+SLOT="0"
+IUSE="doc examples"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+RESTRICT="strip"
+
+RDEPEND="${PYTHON_DEPS}"
+DEPEND="${RDEPEND}
+	>=dev-lang/nasm-2.14.02
+	>=sys-power/iasl-20160729
+	doc? ( app-arch/unzip )"
+
+DOCDIR="${WORKDIR}/Documents"
+HTML_DOCS="${DOCDIR}"/*
+
+# Generated libs for EFI can contain WX sections
+QA_EXECSTACK="
+	usr/lib*/libBaseLib.a:*
+	usr/lib*/libBaseIoLibIntrinsic.a:*
+"
+
+pkg_setup() {
+	python_setup 'python2.7'
+
+	local uname_arch=$(uname -m | sed -e 's:i[3456789]86:IA32:')
+	if [[ ${uname_arch} == "x86_64" ]] || [[ ${uname_arch} == "amd64" ]] ; then
+		ARCH=X64
+	else
+		ARCH=${uname_arch}
+	fi
+
+	# Select toolchain within predefined ones
```

Comment:
Doesn't upstream have definitions for any newer versions? To be honest, I think you should remove all those version checks because we don't really support GCC < 6 anymore; not to mention clang < 5.

Reply:
No newer definition on upstream. I'll remove the unsupported versions.